adr/ADR-43.md

```diff
@@ -35,6 +35,8 @@ Setting the header `Nats-No-Expire` to `1` will result in a message that will ne
 A TTL of zero will be ignored, any other unparsable value will result in a error reported in the Pub Ack and the message
 being discarded.
 
+When a message with the `Nats-TTL` header is published to a stream with the feature disabled the message will be rejected with an error.
```

Comment:
Won't that be a potential footgun?

Publishing to 2.10 with `Nats-TTL` header would be ignored,
in 2.11 and disabled - error

Though of course always ignoring it does not feel right either.

Reply:
Yeah at first I didn’t agree with this idea - still not sure it’s the best but it seems right for now.